python/tank/api.py

```diff
@@ -592,10 +594,38 @@ def preview_filesystem_structure(self, entity_type, entity_id, engine=None):
                                                       engine)
         return folders
 
+    def get_site_cache_root(self):
+        """
+        Returns the location of the site cache root for this Tank instance.
+        """
+        # Invoke the global method which is generic and can compute any site
+        # cache root.
+        global get_site_cache_root
```

Comment:
To access the global version that takes the context (base_url) as a parameter. Is there a cleaner way to call a method with the same name in the global scope?